### DISCUSSION TITLE
GPUs with more than 16MB VRAM

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#4491] [Ideas]
The fastest GPU in 86Box is the Voodoo 3 3500TV and it has only 16MB of VRAM.
Can you implement 32MB and 64MB GPUs.

**Comment by TC1995** [comment]
They're too powerful.

**Comment by Flo9696** [comment]
Would it not be possible to add more VRAM to Voodoo 3 3500TV even if it did not exist in the real? Or can the driver not handle it?
